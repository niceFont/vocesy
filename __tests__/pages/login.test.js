
/*eslint no-undef: off*/
/*eslint camelcase: off*/
import renderer from "react-test-renderer"
import React from "react"
import Login from "../../pages/login"


describe("Login", () => {
    
	it("should render", () => {
		const tree = renderer.create(<Login></Login>)
		expect(tree.toJSON()).toMatchSnapshot()
	})
    
})
