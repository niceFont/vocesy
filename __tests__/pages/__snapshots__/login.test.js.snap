// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render 1`] = `
<div
  className="container"
  style={
    Object {
      "marginBottom": 200,
      "marginTop": 100,
    }
  }
>
  <div
    className="justify-content-center row"
  >
    <div
      className="text-center col-md-4"
    >
      <h5
        style={
          Object {
            "fontWeight": 600,
          }
        }
      >
        Login
      </h5>
    </div>
  </div>
  <div
    className="justify-content-center row"
  >
    <div
      className="col-lg-6 col-md-8 col-sm-10"
      style={
        Object {
          "backgroundColor": "white",
        }
      }
    />
  </div>
  <div
    className="justify-content-center row"
  >
    <div
      className="col-lg-6 col-md-8 col-sm-4"
      style={
        Object {
          "backgroundColor": "white",
          "border": "1px solid lightgray",
          "margin": "0 5% 0 5%",
          "padding": "10%",
        }
      }
    >
      <form
        className=""
        onSubmit={[Function]}
      >
        <div
          className="form-group"
        >
          <label
            className="form-label"
          >
            Username:
          </label>
          <input
            className="form-control"
            onChange={[Function]}
            placeholder="Enter your Username..."
            required={true}
            type="username"
          />
        </div>
        <div
          className="form-group"
        >
          <label
            className="form-label"
          >
            Password:
          </label>
          <input
            className="form-control"
            onChange={[Function]}
            placeholder="Enter your Password..."
            required={true}
            type="password"
          />
        </div>
        <div
          className="form-group"
        >
          <button
            className="btn btn-dark btn-block"
            disabled={true}
            type="submit"
          >
            Login
          </button>
        </div>
        <a
          href="/reset-password"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Forgot password?
        </a>
      </form>
    </div>
  </div>
</div>
`;
